generation/framework/config.py

```diff
@@ -192,8 +192,37 @@
 }
 BINS["dijets"] = {k: dict(v, ident=k) for k,v in BINS["zjet"].items()}
 
-BINS["YB0"] = {k: v for k,v in BINS["zjet"].items() if "YB_00_05" in k}
-BINS["YS0"] = {k: v for k,v in BINS["zjet"].items() if "YS_00_05" in k}
+BINS["zjet_RFUE_all"] = {
```

Comment:
Maybe add a comment what the code below does. It is not really obvious on the first look.

Reply:
done